app/models/miq_schedule/import_export.rb

```diff
@@ -57,11 +57,13 @@ def handle_miq_report_attributes_for_import(miq_schedule)
       miq_schedule
     end
 
-    def import_from_hash(miq_schedule, _options = nil)
+    def import_from_hash(miq_schedule, options = nil)
```

Comment:
Don't think it makes sense to call with `import_from_hash(options[:import_yaml], options)`

```suggestion
    def import_from_hash(miq_schedule, input_userid)
```

Or you can call the variable `userid`. I was using your naming scheme.
I like the fact that `userid` can now be `nil`. Checking for whether you should lookup the user is much simpler than comparing to some hard coded strings (i.e.: `"admin"`)

Reply:
as I said https://github.com/ManageIQ/manageiq/pull/19322/files#r328133148